plugins/hypervisors/vmware/src/main/java/com/cloud/storage/resource/VmwareStorageProcessor.java

```diff
@@ -913,9 +914,12 @@ private String cloneVMforVvols(VmwareContext context, VmwareHypervisorHost hyper
         if (volume.getVolumeType() == Volume.Type.DATADISK)
             vmName = volume.getName();
         if (!_fullCloneFlag) {
+            if (_diskProvisioningStrictness) {
+                throw new CloudRuntimeException("Unable to create linked clones with strict disk provisioning enabled");
```

Comment:
is this applicable for thin and thick disk provisioning types?

Reply:
Fixed